packages/discovery-provider/ddl/functions/handle_track.sql

```diff
@@ -28,12 +28,13 @@ begin
   insert into aggregate_user (user_id) values (new.owner_id) on conflict do nothing;
 
   update aggregate_user
-  set track_count = (
-    select count(*)
+  set (track_count, total_track_count) = (
+    select
+      count(*) filter (where t.is_unlisted = false),
```

Comment:
woah TIL about this syntax.

Reply:
I also TIL'd this syntax. But it's a SQL spec thing!